src/wallet/api/wallet.cpp

```diff
@@ -1161,6 +1225,16 @@ AddressBook *WalletImpl::addressBook() const
     return m_addressBook;
 }
 
+Subaddress *WalletImpl::subaddress() const
```

Comment:
Exactly where should it be const? I just followed the convention in other functions like `history()` and `addressBook()`. Why should only this function be different?